### ISSUE TITLE
can smpl's beta directly be used in STAR?

### ISSUE THREAD

**Issue by wells-wei-wei** [ahmedosman/STAR#5]
I have just used same beta parameters in SMPL and STAR, but they are totally different. whether I use it in a wrong way or not?
 
smpl_para=joblib.load("20200901_zhh_y_t.pkl")
device = torch.device('cuda')
for i in tqdm(range(len(smpl_para[1]['frame_ids']))):
    pose=smpl_para[1]['pose'][i]
    pose=pose[np.newaxis, :]
    beta=smpl_para[1]['betas'][i]
    beta=beta[np.newaxis, :]

    poses = torch.cuda.FloatTensor(pose)
    poses = Variable(poses,requires_grad=True)
    betas = torch.cuda.FloatTensor(beta)
    betas = Variable(betas,requires_grad=True)
    trans = torch.cuda.FloatTensor(np.zeros((1,3)))
    trans = Variable(trans,requires_grad=True)
    d = star(poses, betas, trans)

    f="%06d" % i

    vertices=d.detach().cpu().numpy().squeeze()
    faces=d.f
    out_mesh = trimesh.Trimesh(vertices, faces)
    out_mesh.export("star_obj/"+f+".obj")

**Comment by ahmedosman** [comment]
Hi @wells-wei-wei , both STAR pose and shape space are different from SMPL, so we can't plug-in the SMPL parameters in STAR. I will commit a convertor script to convert from SMPL to STAR. Thanks,Ahmed

**Comment by ahmedosman** [comment]
Hi @wells-wei-wei  I just pushed a convertor script that given 'smpl meshes' returns and save the star parameters. It's under https://github.com/ahmedosman/STAR/blob/master/convertors/convert_smpl_to_star.py

**Comment by leesky1c** [comment]
Is there a way to regress the STAR parameters directly like HMR or VIBE? Thanks.

**Comment by ahmedosman** [comment]
Hi @leeskyed, one would need to retrain the networks (HMR and VIBE) with STAR.  At the moment, there isn't a direct regressor of parameters STAR from images/videos.
